<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MainMenuWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1058</height>
    </rect>
   </property>
   <widget class="QWidget" name="simulatorWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>1291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="LoadSimulator" stretch="0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="simulatorPathBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the path to weiss schwarz main folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Input path to Weiss Schwarz Simulator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadSimulatorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load the simulator sets from the given path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MenusWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>1920</width>
      <height>1058</height>
     </rect>
    </property>
    <widget class="QLabel" name="MainMenuTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>1911</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>96</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Weiss SimulatorTool</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="MainMenuCardListButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>310</y>
       <width>871</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Card List Visualization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MainMenuQuitButton">
     <property name="geometry">
      <rect>
       <x>1260</x>
       <y>810</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="unloadButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>820</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unload Simulator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenImportWidgetButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>610</y>
       <width>871</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Import From Encore Decks</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MainMenuDeckEditorButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>460</y>
       <width>871</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Simulator Deck Editor</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
